$NetBSD$

A few things are done here:

1) Make libm, libz, libjansson and libiconv dynamically linked, instead of statically.
2) Use system libjansson instead of the one included with zero-epwing.
3) Link libintl if |UNIX AND NOT APPLE|.
4) Specify the full path to the libeb static library, since (at least on macOS) the linker
   has trouble linking it otherwise.

--- CMakeLists.txt.orig	2017-10-06 02:15:49.000000000 +0000
+++ CMakeLists.txt
@@ -1,10 +1,15 @@
 cmake_minimum_required(VERSION 3.5)
 project(zero-epwing)
 include_directories(eb)
-link_directories(eb/eb/.libs jansson/lib)
+link_directories(eb/eb/.libs)
 add_executable(zero-epwing main.c book.c convert.c hooks.c)
-target_link_libraries(zero-epwing libm.a libeb.a libm.a libz.a libjansson.a)
+target_link_libraries(zero-epwing
+    m z jansson
+    ${CMAKE_SOURCE_DIR}/eb/eb/.libs/libeb.a
+)
+if (UNIX AND NOT APPLE)
+    target_link_libraries(zero-epwing intl)
+endif (UNIX AND NOT APPLE)
 if (WIN32 OR APPLE)
-    target_link_libraries(zero-epwing libiconv.a)
+    target_link_libraries(zero-epwing iconv)
 endif (WIN32 OR APPLE)
-target_link_libraries(zero-epwing)
